test/subscription_map.cpp

```diff
@@ -262,6 +286,9 @@ BOOST_AUTO_TEST_CASE( test_multiple_subscription_modify ) {
         value.const_mem_fun();
         value.non_const_mem_fun();
     });
+
+
+
```

Comment:
bunch of extra newlines?

Reply:
@kleunen , could you remove the newlines ?